Automattic/jetpack
# CRM: Update JPCRM Screenshots on WordPress.org

## Proposed changes:
* This PR updates Jetpack CRM's screenshots on WordPress.org.

### Other information:


## Jetpack product discussion
https://github.com/Automattic/zero-bs-crm/issues/3161

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
Check if the update images are good for our screenshot section on our WordPress.org plugin page.
